ext/mysqli/tests/bug81335.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+Bug #81335: Packets out of order after connection timeout
+--EXTENSIONS--
+mysqli
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
+
+if (!defined('MYSQLI_STORE_RESULT_COPY_DATA')) die('skip requires mysqlnd');
+
+require_once 'connect.inc';
+if (!$link = @my_mysqli_connect($host, $user, $passwd, $db, $port, $socket)) die("skip cannot connect");
+if (mysqli_get_server_version($link) < 80024) {
```

Comment:
This is an error message that does not occur with MariaDB, so please skip it if using MariaDB.

```suggestion
if (mysqli_get_server_version($link) < 80024 || str_contains(mysqli_get_server_info($link), 'MariaDB')) {
```

Reply:
thanks